doc/source/deploy-guide/prepare-localhost.rst

```diff
@@ -138,28 +194,30 @@ the following example if you are running on engcloud (SUSE employees):
      engcloud:
        region_name: CustomRegion
        auth:
-         auth_url: https://engcloud.prv.suse.net:5000/v3
+         auth_url: https://keystone_url/v3
          username: foctodoodle # your username here
          password: my-super-secret-password # your password here or add it into secure.yaml
          project_name: cloud
          project_domain_name: default
-         user_domain_name: ldap_users
+         user_domain_name: ldap_users # this is just an example, adapt to your needs
```

Comment:
All values are examples, right? I'm not sure if it makes sense to specify right here